lib/utils/refreshTimer.test.ts

```
@@ -0,0 +1,39 @@
+    const callback = vi.fn();
+    expect(() => RefreshTimer.setRefreshTimer(-10, callback)).toThrowError(
+      "Timer duration must be positive",
+    );
+  });
+
+  it("should throw error when window is undefined", () => {
+    const originalWindow = global.window;
+    // Temporarily delete the window object
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    delete global.window;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)